config/routes.rb

```diff
@@ -42,6 +42,20 @@
     end
 
     namespace :api, defaults: { format: "json" } do
+      # API V1 is in pre-release: Available iff api_v1 FeatureFlag is enabled
```

Comment:
Yes, all routes should exist in both of them, so they'll be duplicated. This will make `routes.rb` look kind of cluttered... I'm starting to think maybe extracting both of these into a separate route file that gets "drawn" in `routes.rb`, kind of like [what we do with admin routes](https://github.com/forem/forem/blob/main/config/routes.rb#L28) that have a separate [`/config/routes/admin.rb` file](https://github.com/forem/forem/blob/main/config/routes/admin.rb). Any other ideas/suggestions are welcome though 😄

Reply:
That sounds great, I'll definitely give this a try @jeremyf! 👍🏼